www/class/centreonAuth.SSO.class.php

```diff
@@ -586,21 +601,26 @@ public function logout(
         string $clientId,
         string $clientSecret,
         string $refreshToken,
+        bool $clientBasicAuth,
```

Comment:
can you also modify function definition?

Reply:
Sorry, added the function documentation for this parameter